### PR TITLE
Allow to set the frequencies to use for the R-Crit computation.

### PR DESCRIPTION
**Pull request by SPalominos**
Allow to set the frequencies to use for the R-Crit computation.

### CODE REVIEW

File: noisemodelling-pathfinder/src/main/java/org/noise_planet/noisemodelling/pathfinder/CnossosPropagationData.java [hunk 1]
```diff
@@ -119,6 +119,14 @@
 
     Map<String, Integer> sourceFieldNames = new HashMap<>();
 
+    public List<Integer> freq_lvl = List.of(63, 125, 250, 500, 1000, 2000, 4000, 8000);
```
Comment (nicolas-f) [inline noisemodelling-pathfinder/src/main/java/org/noise_planet/noisemodelling/pathfinder/CnossosPropagationData.java:122]: Use
```java
Arrays.asList(PropagationProcessPathData.asOctaveBands(PropagationProcessPathData.DEFAULT_FREQUENCIES_THIRD_OCTAVE))
```

**Review by nicolas-f — CHANGES_REQUESTED** [review]

File: noisemodelling-pathfinder/src/main/java/org/noise_planet/noisemodelling/pathfinder/CnossosPropagationData.java (hunk 1 above)
Comment (nicolas-f) [reply noisemodelling-pathfinder/src/main/java/org/noise_planet/noisemodelling/pathfinder/CnossosPropagationData.java:122]: As the user may have sound source in third octave band or special frequencies bands

Comment (SPalominos) [reply noisemodelling-pathfinder/src/main/java/org/noise_planet/noisemodelling/pathfinder/CnossosPropagationData.java:122]: Done, thanks!

Comment (SPalominos) [reply noisemodelling-pathfinder/src/main/java/org/noise_planet/noisemodelling/pathfinder/CnossosPropagationData.java:122]: Done, thanks!

**Comment by nicolas-f** [comment]
Ok, now you have to remove the migrated code on PropagationProcessPathData (avoid duplication of code)

**Comment by pierromond** [comment]
Ok to merge

**Review by pierromond — APPROVED** [review]

**Review by nicolas-f — APPROVED** [review]

**Pull request merged**